pagopa/pn-frontend
# PN-13433 - Mobile stepper of PnWizard

## Short description
Implement mobile stepper of PnWizard component

## List of changes proposed in this pull request
- Create PnWizardStepper

## How to test
Start PF and navigate to `/test` page. Then resize the windows and check that mobile stepper is shown
